[2024-12-12T08:08:01.732+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-12T08:08:01.755+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_orchestrator.dbt_run_fact.dbt_run_fact_model_fact_sales manual__2024-12-12T08:05:08.618053+00:00 [queued]>
[2024-12-12T08:08:01.765+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_orchestrator.dbt_run_fact.dbt_run_fact_model_fact_sales manual__2024-12-12T08:05:08.618053+00:00 [queued]>
[2024-12-12T08:08:01.765+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-12T08:08:01.779+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run_fact.dbt_run_fact_model_fact_sales> on 2024-12-12 08:05:08.618053+00:00
[2024-12-12T08:08:01.785+0000] {standard_task_runner.py:63} INFO - Started process 250 to run task
[2024-12-12T08:08:01.789+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_orchestrator', 'dbt_run_fact.dbt_run_fact_model_fact_sales', 'manual__2024-12-12T08:05:08.618053+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpc7_8hsh1']
[2024-12-12T08:08:01.790+0000] {standard_task_runner.py:91} INFO - Job 143: Subtask dbt_run_fact.dbt_run_fact_model_fact_sales
[2024-12-12T08:08:01.832+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_orchestrator.dbt_run_fact.dbt_run_fact_model_fact_sales manual__2024-12-12T08:05:08.618053+00:00 [running]> on host 34fdc836a0bb
[2024-12-12T08:08:01.918+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ma7moud' AIRFLOW_CTX_DAG_ID='dbt_orchestrator' AIRFLOW_CTX_TASK_ID='dbt_run_fact.dbt_run_fact_model_fact_sales' AIRFLOW_CTX_EXECUTION_DATE='2024-12-12T08:05:08.618053+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-12T08:05:08.618053+00:00'
[2024-12-12T08:08:01.919+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-12T08:08:01.931+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-12T08:08:01.932+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/ecommerce --models models/olap_model/fact/fact_sales']
[2024-12-12T08:08:01.945+0000] {subprocess.py:86} INFO - Output:
[2024-12-12T08:08:04.451+0000] {subprocess.py:93} INFO - [0m08:08:04  Running with dbt=1.8.0
[2024-12-12T08:08:04.694+0000] {subprocess.py:93} INFO - [0m08:08:04  [[33mWARNING[0m]: Deprecated functionality
[2024-12-12T08:08:04.695+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-12-12T08:08:04.696+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-12-12T08:08:04.696+0000] {subprocess.py:93} INFO - information.
[2024-12-12T08:08:04.923+0000] {subprocess.py:93} INFO - [0m08:08:04  Registered adapter: postgres=1.8.0
[2024-12-12T08:08:05.402+0000] {subprocess.py:93} INFO - [0m08:08:05  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-12-12T08:08:05.402+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2024-12-12T08:08:05.402+0000] {subprocess.py:93} INFO - - models.ecommerce.staging.fact
[2024-12-12T08:08:05.402+0000] {subprocess.py:93} INFO - - models.ecommerce.staging.dimensions
[2024-12-12T08:08:05.742+0000] {subprocess.py:93} INFO - [0m08:08:05  Found 15 models, 1 snapshot, 112 data tests, 12 sources, 430 macros
[2024-12-12T08:08:05.747+0000] {subprocess.py:93} INFO - [0m08:08:05  The selection criterion 'models/olap_model/fact/fact_sales' does not match any enabled nodes
[2024-12-12T08:08:05.749+0000] {subprocess.py:93} INFO - [0m08:08:05
[2024-12-12T08:08:05.750+0000] {subprocess.py:93} INFO - [0m08:08:05  Nothing to do. Try checking your model configs and model specification args
[2024-12-12T08:08:06.754+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-12T08:08:06.756+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-12T08:08:06.784+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_orchestrator, task_id=dbt_run_fact.dbt_run_fact_model_fact_sales, run_id=manual__2024-12-12T08:05:08.618053+00:00, execution_date=20241212T080508, start_date=20241212T080801, end_date=20241212T080806
[2024-12-12T08:08:06.819+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-12T08:08:06.844+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-12T08:08:06.846+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
